File: Python/abc197_a/main.py
def solve(s):
    head = s[0:1]
    tail = s[1:]
    return tail + head


def get_input():
    return input()


def main():
    # case: get input to int nums
    # a, b, c = map(int, input().split())
    inpt = get_input()
    print(solve(inpt))


if __name__ == "__main__":
    main()

File: C/abc197_a/main.c
#include <stdio.h>
#include <string.h>

void solve(const char *input, char *answer) {
    char head = input[0];
    char tail[3];

    strncpy(tail, &input[1], 2);
    tail[2] = '\0';

    strncpy(answer, &tail[0], 2);
    answer[2] = head;
    answer[3] = '\0';
}

int main(void) {
    char input[4];
    char answer[4];

    scanf("%3s", input);

    solve(input, answer);

    printf("%s\n", answer);
    return 0;
}
